src/dpi.rs

```diff
@@ -58,7 +58,7 @@
 //! DPI settings. This gives you a chance to rescale your application's UI elements and adjust how
 //! the platform changes the window's size to reflect the new scale factor. If a window hasn't
 //! received a [`ScaleFactorChanged`](crate::event::WindowEvent::ScaleFactorChanged) event,
-//! then its scale factor is `1.0`.
+//! then its scale factor is [[window.scale_factor()]].
```

Comment:
```suggestion
//! then its scale factor can be found my calling [[window.scale_factor()]].
```
I feel like this reads better.